frontend/Auth/FinishResetPassword.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { Button, ButtonKind } from "../Styleguide/Button";
+import { FormField } from "../Styleguide/FormField";
+import { FormFieldLabel } from "../Styleguide/FormFieldLabel";
+import { Input } from "../Styleguide/Input";
+import { RouterProps } from "react-router";
+import { useMutation, useQuery } from "react-query";
+import { flaxFetch } from "../Utils/flaxFetch";
+import { unary } from "lodash-es";
+
+export function FinishResetPassword(props: RouterProps) {
+  const [finishResetPasswordFormData, setFinishResetPasswordFormData] =
+    useState<FinishResetPasswordFormData>({
+      password: "",
+    });
+
+  const paramString = props.history.location.search;
+  let searchParams = new URLSearchParams(paramString);
+  const token = searchParams.get("jwt") || "MISSING TOKEN";
+
+  const queryFunctionhelper = (queryKey) => {
+    const token = queryKey["queryKey"][0];
+    return flaxFetch<object>(
+      `/validate-token/${token}?tokenType=passwordReset`,
+      {
+        method: "GET",
+      }
+    );
+  };
+
+  const info = useQuery(token, queryFunctionhelper);
```

Comment:
Info isn't a very good variable name because it's so vague. Maybe `tokenQuery`? And then `tokenQuery.data` would be the response data

Reply:
Completely agree.  Will change to `tokenValidationResponse`.